.clang-format

```diff
@@ -89,7 +89,7 @@ ColumnLimit: 160
 CommentPragmas: '^ IWYU pragma:'
 CompactNamespaces: false
 ConstructorInitializerIndentWidth: 4
-ContinuationIndentWidth: 4
+ContinuationIndentWidth: 8
```

Comment:
this in particular improves the readability of constructors (and seems to be already used in existing code), as for example:

with value of 4:
```
my_class::my_class(type1 param1, type2 param2,
    type3 param3, type4 param4)
    : _member1(param1), _member2(param2)
    , _member3(param3), _member4(param4) {
    // The constructor body
}
```

with value of 8:
```
my_class::my_class(type1 param1, type2 param2,
        type3 param3, type4 param4)
    : _member1(param1), _member2(param2)
    , _member3(param3), _member4(param4) {
    // The constructor body
}
```

the second case looks a bit better as it more clearly separates the parameters and the initializers

Reply:
In fact the [Seastar coding style](https://github.com/scylladb/seastar/blob/master/coding-style.md) even specifies that the continuation lines should have double indent:

> If a line becomes excessively long (>160 characters?), or is just complicated, break it into two or more lines. The second (and succeeding lines) are _continuation lines_, and **have a double indent**:
> ...